packages/plugins/documentation/preview-output/verbose/models/Comment.md

```
@@ -0,0 +1,223 @@
+<a id="access-policies"></a>
+
+## 🔐 Access Policies
+
+> ZenStack access control rules that determine who can read, create, update, or delete records.
+
+> [!IMPORTANT]
+> Operations are **denied by default**. `@@allow` rules grant access; `@@deny` rules override any allow.
+
+| Operation | Rule | Effect |
+| --- | --- | --- |
+| read | `article.status == 'PUBLISHED' || workspace.members?[user == auth()]` | Allow |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)